CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to
 
 ### Added
 
```

Comment:
Plus this panel in the Video Dashboard

Reply:
I think this is already present in the changelog)